enrolments/tests/views/test_class.py

```diff
@@ -33,6 +33,12 @@ def setUp(self):
             facilitator_id=self.user.id,
             attendance=[{"date": "2020-01-01", "attendees": [self.student1.id]}],
         )
+        self.class_create = Class.objects.create(
+            name="Test Class Create",
+            days=[Class.MONDAY, Class.WEDNESDAY],
+            location="Waterloo",
+            facilitator=self.user,
+        )
```

Comment:
is this being used anywhere? :o

Reply:
oh yikes good catch lol, i meant to use it in `test_create_class`